test_files/values.alp

```diff
@@ -15,6 +18,20 @@ let test_atom = :nope
 let test_bool = true
 let test_chars = c"x"
 let test_tuple = (100, "100")
+let test_unit = ()
+
+let wrapped_int = AnInt 41
```

Comment:
I'm fine with merging this. The purpose of this PR was to add `chars`and `()` as basic types. I just happened to notice the described behavior when implementing this and wondered if it was intentional.